src/main/resources/application.yml

```diff
@@ -141,3 +143,4 @@ application:
     poll:
       frequency: 60000
       initialDelay: 2000
+      lockAtLeastFor: PT10S
```

Comment:
should the be -1 to make it consistent with now?

Reply:
In the previous version of SchedulerConfiguration it was:
@EnableSchedulerLock(defaultLockAtLeastFor = "PT10S", defaultLockAtMostFor = "PT20M")
So I just matched it with that